joomla/joomla-cms
# [4.3][Guided Tours] Hide 'Take a tour' on small devices

Addresses issue #40011.

The guided tours are accessible throughout the administration.
However, on small devices, the tours cannot always run properly because some of the content is hidden from view and from the tours (targets are not found so the steps end up in the upper left corner of the window).

I am not sure this is the best approach, so please comment.

### Summary of Changes

Add CSS to hide the tour dropdown on small viewports.

### Testing Instructions

Make sure you see the 'Take a tour' button on the top toolbar of the administrator console.
Resize the window down. Around 576px, the dropdown should disappear.

### Actual result BEFORE applying this Pull Request

You can take a tour on any screen size.

### Expected result AFTER applying this Pull Request

'Take a tour' is not visible if the viewport is small or you are on a mobile device (tablets should be fine, but usually not smartphones).

This does not prevent the user to start a tour, resize the window down and continue the tour with possible 'failure' (no error or anything, just unexpected behavior in case an element is not 'visible').

### Link to documentations
Please select:
